packages/experimental-nextjs-app-support/src/index.shared.ts

```diff
@@ -25,3 +27,19 @@ export class ApolloClient<
    */
   static readonly info = bundle;
 }
+
+/**
+ * A version of `InMemoryCache` to be used with streaming SSR.
+ *
+ * For more documentation, please see {@link https://www.apollographql.com/docs/react/api/cache/InMemoryCache | the Apollo Client API documentation}.
+ *
+ * @public
+ */
+export class InMemoryCache extends UpstreamInMemoryCache {
```

Comment:
We add our own implementation here to ensure the error messages mention this package, not the parent package.